src/main/java/com/odysseusinc/arachne/portal/service/impl/BaseArachneSecureServiceImpl.java

```diff
@@ -260,6 +269,15 @@ public boolean checkDataNodeAdmin(ArachneUser user, DataNode dataNode) {
         return getRolesByInsight(user, insight);
     }
 
+    @Override
+    public List<ParticipantRole> getRolesByOrganization(ArachneUser user, Organization organization) {
+
+        final User standardUser = userRepository.findByEmailAndEnabledTrue(user.getUsername());
+        final boolean admin = standardUser.getRoles().stream().anyMatch(r -> r.getName().equals("ROLE_ADMIN"));
```

Comment:
Instead of ROLE_ADMIN you can use com.odysseusinc.arachne.portal.service.BaseRoleService#ROLE_ADMIN